message-service/src/main/java/sjchat/messages/MessageService.java

```diff
@@ -140,6 +115,11 @@ public void getChat(GetChatRequest req, StreamObserver<GetChatResponse> response
 
   @Override
   public void createChat(CreateChatRequest req, StreamObserver<CreateChatResponse> responseObserver) {
+    if (req.getTitle() == null || req.getTitle().length() == 0) {
+      responseObserver.onError(new Exception("Title is empty or missing"));
+      responseObserver.onCompleted();
+    }
```

Comment:
Checking if the title is null is actually redundant. It's not possible to do a grpc request with null as any of the data members. That would throw a NullPointerException at the caller